boards/arm/nrf5340_dk_nrf5340/Kconfig

```diff
@@ -5,6 +5,21 @@
 
 if BOARD_NRF5340_DK_NRF5340_CPUAPP || BOARD_NRF5340_DK_NRF5340_CPUAPPNS
 
+config BOARD_ENABLE_DCDC_APP
+	bool "Enable Application Core DCDC converter"
```

Comment:
Please, use "MCU" instead of "Core" ; this is the name we use consistently in doc/index.rst.

Reply:
and in similar places below